lib/srv/uacc/uacc.h

```diff
@@ -39,7 +39,7 @@ int UACC_UTMP_PATH_DOES_NOT_EXIST = 8;
 // We use this string pointer to store a potential error message from glibc in certain cases.
 //
 // At first glance this may seem racy, however this pointer is protected by the mutex that wraps all uacc logic on the Go side.
-char* uaccPathErr;
+// char* uaccPathErr;
```

Comment:
I think we should remove this commented-out global along with the comment above it.